apps/src/turtle/blocks.js

```diff
@@ -1126,7 +1126,7 @@ exports.install = function (blockly, blockInstallOptions) {
 
   generator.sticker = generator.turtle_stamp = function () {
     return 'Turtle.drawSticker("' + this.getTitleValue('VALUE') +
-        '", \'block_id_' + this.id + '\');\n';
+        '", null, \'block_id_' + this.id + '\');\n';
```

Comment:
this doesn't change the actual block right?

Reply:
Correct, just the JS code generated from the block.

Reply:
Correct, it doesn't change the block.  The next iteration of this project is to create a block that has a size option.